src/deploy.js

```diff
@@ -12,6 +12,14 @@ async function deploy() {
 
 	const name = slugify(GITHUB_REPOSITORY_NAME);
 
+	console.log('Final Object', {
```

Comment:
do we need to keep these logs in final version?

Reply:
No, we don't need them, just added to verify that configs were being passed.